config.py

```diff
@@ -7,23 +7,15 @@
 # Application settings
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
-API_DESC = 'An API for serving models'
+API_TITLE = 'MAX Facial Recognizer'
+API_DESC = 'Detect faces from the image and generate embeddings for the detected faces'
```

Comment:
```suggestion
API_DESC = 'Recognize faces in an image and extract embedding vectors for each face.'
```
per the description here https://developer.ibm.com/exchanges/models/all/max-facial-recognizer/